lib/lambda_ethereum_consensus/state_transition/operations.ex

```diff
@@ -8,13 +8,103 @@ defmodule LambdaEthereumConsensus.StateTransition.Operations do
 
   alias SszTypes.{
     Attestation,
+    BeaconBlock,
+    BeaconBlockHeader,
     BeaconState,
     ExecutionPayload,
     SyncAggregate,
     Validator,
     Withdrawal
   }
 
+  @spec process_block_header(BeaconState.t(), BeaconBlock.t()) ::
+          {:ok, BeaconState.t()} | {:error, String.t()}
+  def process_block_header(
+        %BeaconState{slot: state_slot, latest_block_header: latest_block_header} = state,
+        %BeaconBlock{slot: block_slot, proposer_index: proposer_index, parent_root: parent_root} =
+          block
+      ) do
+    with :ok <- check_slots_match(state_slot, block_slot),
+         :ok <-
+           check_block_is_newer_than_latest_block_header(block_slot, latest_block_header.slot),
+         :ok <- check_proposer_index_is_correct(proposer_index, state),
+         :ok <- check_parent_root_match(parent_root, latest_block_header),
+         {:ok, state} <- cache_current_block(state, block) do
+      # Verify proposer is not slashed
+      proposer = state.validators |> Enum.fetch!(proposer_index)
+
+      case proposer.slashed do
```

Comment:
nit: change for if & else

Reply:
Done.

Please take a look at the lint error. I don't seem to understand it.

Reply:
It seems there's a space at the end of comments in lines 69, 82, and 95 of `lib/lambda_ethereum_consensus/state_transition/operations.ex`